cg/cli/demultiplex/demux.py

```diff
@@ -220,3 +228,16 @@ def confirm_flow_cell_sync(context: CGConfig, source_directory: str):
                 source_flow_cell.name,
                 DemultiplexingDirsAndFiles.COPY_COMPLETE,
             ).touch()
+
+
+@click.command(name="create-manifest-files")
+@click.option(
+    "-s",

```

Comment:
If it is not one of the unix standards like `-v, -h`. I think we can skip a short flag - you never remember them and should always use the long name in code.
```suggestion
```